docs/userGuide/syntax/annotations.md

```diff
@@ -0,0 +1,110 @@
+  <!-- Default Trigger (click)-->
+  <a-point x="33%" y="50%" content="Lorem ipsum dolor sit amet" />
+  <!-- Set Trigger to hover focus -->
+  <a-point x="66%" y="50%" content="Lorem ipsum dolor sit amet" trigger="hover focus"/>
+</annotate>
+</variable>
+</include>
+
+<br>
+
+**Displaying content as legends in Annotate Point** <br>
+
+Annotate point also allows users to display its content as a legend below the diagram. However, this only works when a label is provided for the point.
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+
+<annotate src="https://www.researchgate.net/profile/Chaiwat-Sakul/publication/228949121/figure/fig2/AS:300654661783574@1448693065750/The-previous-square-root-circuit-1.png" width="500">
+  <!-- Default Legend (popover only)-->
+  <a-point x="25%" y="50%" content="There is only text when you click me" label="1"/>
+  <!-- Set Legend to bottom only -->
+  <a-point x="50%" y="50%" content="Clicking on this does nothing" label="2" legend="bottom"/>
+  <!-- Set Legend to both -->
+  <a-point x="75%" y="50%" content="There is text at both locations" header="Headers are displayed as well" label="3" legend="both"/>
+</annotate>
+</variable>
+</include>
+
+<br>
+
+****`<a-point>` Options****
+
+| Name      | Type     | Default   | Description                                                                                                               |
+| --------- | -------- | --------- | ------------------------------------------------------------------------------------------------------------------------- |
+| x         | `String` | `''`      | How to position the x-coordinate of the point.<br>Supports range of values from `0%` to `100%`.                           |
+| y         | `String` | `''`      | How to position the y-coordinate of the point.<br>Supports range of values from `0%` to `100%`.                           |
+| content      | `String` | `''`      | The content to be inserted into the body of the text.<br>The body will be omitted if this is not provided                 |
+| header    | `String` | `''`      | The content to be inserted into the header of the text.<br>The header will be omitted if this is not provided             |
+| trigger   | `String` | `click`   | How the Popover is triggered.<br>Supports: `click`, `focus`, `hover`, or any space-separated combination of these.        |
+| placement | `String` | `top`     | How to position the Popover.<br>Supports: `top`, `left`, `right`, `bottom`.                                               |
+| label     | `String` | `''`      | The content to be shown over the point.<br>The label will be omitted if this is not provided                              |
+| size      | `String` | `'40'`    | The size of the point in pixels.                                                                                          |
```

Comment:
Since the size of the image is responsive (the width and height changes with the screen size), was wondering if we should support more dynamic sizes like `%` or `vw` as well so that it scales with the screen size. Using pixel will fixed the size of the annotation and it may become too big or too small when the image scales.

If it is too much work to implement this we can consider doing it in a follow-up PR as an extension to this :)

Reply:
This sounds like a good idea, I can definitely scale it to the size of the image. I will add this feature within this PR.